src/EditorFeatures/CSharpTest2/Recommendations/WhenKeywordRecommenderTests.cs

```diff
@@ -66,409 +66,285 @@ public class WhenKeywordRecommenderTests : KeywordRecommenderTests
 
         [Fact, Trait(Traits.Feature, Traits.Features.KeywordRecommending)]
         [WorkItem(24113, "https://github.com/dotnet/roslyn/issues/24113")]
-        public async Task TestForSwitchCase_AfterDeclarationPattern()
-        {
+        public async Task TestForSwitchCase_AfterDeclarationPattern() =>
```

Comment:
@CyrusNajmabadi I thought this might actually be a pretty good idea because it will force me to only have one test per each test method. What do you think?